Fix yaml provisioning (#137)
kartverket/Regelrett: conf/provisioning/README.md

`metadata`: Contains metadata related to the record’s answer, as well as optional configuration.

`answerMetadata`: Defines how the answer for the record should be handled. Contains the following fields:

- `Type`: Required, select the type of this records answer, select between: `SELECT_MULTIPLE`,`SELECT_SINGLE`, `TEXT_MULTI_LINE`, `TEXT_SINGLE_LINE`, `PERCENT`, `CHECKBOX`, `TIME`

- `Unit`: Optional, List of strings - specefy units of the answers.

- `expiry`: Optional, int - set number of weeks an answer is valid, until it is flagged as expired in Regelrett

`optionalFields`: Defines values for each column in the record.
Each key must correspond to a column name
Each value represents the data for that column in the given record

```yaml
name: "YAML-data" 
columns: 
  - type: "OPTION_SINGLE"  #Choose between OPTION_MULTIPLE, OPTION_SINGLE or TEXT
    name: "Kortnavn" #Column name
  - type: "OPTION_SINGLE"
    name: "ID" #ID columns should always be included in every Schema. 
  - type: "OPTION_SINGLE"
    name: "Kontroller"
  - type: "OPTION_SINGLE"
    name: "Svar"
  - type: "OPTION_SINGLE"
    name: "Priority"

>     options: #Specefies the options for the entire column, meaning you can not overrirde theese options on the record specefications.

Reviewer: ```suggestion
    options: #Specifies the options for the entire column, meaning you can not override these options in the record specifications.
```

      - name: "MÅ" 

>         color: "orangeDark1" #choose a color to go with you option thoriggout the entire column.

Reviewer: ```suggestion
        color: "orangeDark1" #choose a color to go with this option throughout the entire column.
```
